demos/common/python/models/segmentation.py

```diff
@@ -84,3 +84,15 @@ def postprocess(self, outputs, meta):
 
         result = cv2.resize(result, (input_image_width, input_image_height), 0, 0, interpolation=cv2.INTER_NEAREST)
         return result
+
+
+class SalientObjectDetectionModel(SegmentationModel):
+
+    def postprocess(self, outputs, meta):
+        input_image_height = meta['original_shape'][0]
+        input_image_width = meta['original_shape'][1]
+        mask = outputs[self.out_blob_name].squeeze()
```

Comment:
It seems questionable to call this a mask. Masks are normally boolean, while this is an array of real numbers. And it's even called a saliency _map_ in the documentation.

Reply:
renamed